hbase-server/src/main/java/org/apache/hadoop/hbase/ipc/CallRunner.java

```diff
@@ -222,6 +223,7 @@ public void drop() {
       call.setResponse(null, null, CALL_DROPPED_EXCEPTION, "Call dropped, server "
         + (address != null ? address : "(channel closed)") + " is overloaded, please retry.");
       call.sendResponseIfReady();
+      this.rpcServer.getMetrics().exception(CALL_DROPPED_EXCEPTION);
```

Comment:
Where do we call this method in the past? Will this introduce double accounting?

Reply:
As far as I can tell, we do not report them. We added this exception in https://issues.apache.org/jira/browse/HBASE-26623, but the problem is when we drop a call here it doesn't end up hitting any of the other invocations of this method. So the value is always 0 despite throwing these exceptions to clients.

Reply:
Typically this is called in a few places, for example in [RpcServer](https://github.com/apache/hbase/blob/master/hbase-server/src/main/java/org/apache/hadoop/hbase/ipc/RpcServer.java#L456) or in a few places in RSRpcServices. But a couple exceptions need to be handled a level above this. For example CallQueueTooBigException is [handled in the connection](https://github.com/apache/hbase/blob/master/hbase-server/src/main/java/org/apache/hadoop/hbase/ipc/ServerRpcConnection.java#L647) and RequestTooBig in the [netty frame decoder](https://github.com/apache/hbase/blob/master/hbase-server/src/main/java/org/apache/hadoop/hbase/ipc/NettyRpcFrameDecoder.java#L134).

So this exception is another example of a special case like CallQueueTooBig/RequestTooBig, that doesn't get handled by the default try/catches we have in RpcServer/RSRpcServices.